Common/src/main/resources/assets/toolleveling/lang/ru_ru.json

```diff
@@ -0,0 +1,119 @@
+{
+	"block.toolleveling.tool_leveling_table": "Перезачаровыватель",
+	"container.toolleveling.tool_leveling_table": "Перезачаровыватель",
+	"commands.superenchant.failed": "%s не поддерживает это зачарование, вы можете включить его в настройках мода (%s)",
+	"screen.toolleveling.tool_leveling_table.start_upgrade.button_text": "Улучшить",
+	"screen.toolleveling.tool_leveling_table.start_upgrade.button_tooltip": "Запустите процесс улучшения",
+	"screen.toolleveling.tool_leveling_table.success_chance.field_title": "Успех итерации",
+	"screen.toolleveling.tool_leveling_table.success_chance": "Шанс успеха",
+	"screen.toolleveling.tool_leveling_table.success_chance.min": "Минимальный %s%%",
+	"screen.toolleveling.tool_leveling_table.success_chance.max": "Максимальный %s%%",
+	"screen.toolleveling.tool_leveling_table.summary.field_title": "Сводка",
+	"screen.toolleveling.tool_leveling_table.summary.field_text": "Вплоть до %s чар будут наложены на предмет, с уровнем %s",
+	"screen.toolleveling.tool_leveling_table.summary.field_text.multi": "Вплоть до %s чар будут наложены на предмет, с уровнями между %s и %s",
+	"screen.toolleveling.tool_leveling_table.percentages.button_text": "Информация",
+	"screen.toolleveling.tool_leveling_table.percentages.button_tooltip": "Показывать/скрывать проценты",
+	"screen.toolleveling.tool_leveling_table.percentages.field_title": "Выбранные чары",
+	"screen.toolleveling.tool_leveling_table.help.button_text": "Справка",
+	"screen.toolleveling.tool_leveling_table.help.field_title": "Как это работает?",
+	"screen.toolleveling.tool_leveling_table.help.field_text": "1. Положите зачарованный предмет в большой средний слот\n\n2. Поместите достаточное количество зачарованных книг в левые ячейки (не менее %s книг)\n\n3. Книжные полки вокруг стола увеличивают шансы на успех\n\n4. Специальные предметы в нужных ячейках приведут к большему количеству чар более высокого уровня\n\n5. Нажмите 'Улучшить'",
+	"screen.toolleveling.tool_leveling_table.help.button_tooltip": "Показать/скрыть текст справки",
+	"commands.link.click_here": "нажмите сюда",
+	"commands.toolleveling.config.path": "Настройки для %s могут быть найдены здесь: %s",
+	"commands.toolleveling.config.reload": "Настройки были успешно перезагружены.",
+	"commands.toolleveling.config.noconfig": "Настройки для мода не найдены.",
+	"commands.toolleveling.config.info_general": "Общую информацию о настройках можно найти здесь: %s",
+	"commands.toolleveling.config.info_single": "Полное объяснение конфигурации %s и её опций можно найти здесь: %s",
+	"commands.toolleveling.config.reset": "Для настройки %s было установлено значение по умолчанию.",
+	"enchantment.level.0": "0",
+	"enchantment.level.11": "11",
+	"enchantment.level.12": "12",
```

Comment:
Little tip, if you want to have the lines exactly like the english version, you can just erase them.  
The game will always use the english translation of everything if a translation is not found in the selected language.